modules/installation-and-upgrade/pages/container-deployment/uyuni/migrate-uyuni-proxy-from-micro55-tumbleweed.adoc

```diff
@@ -0,0 +1,204 @@
+= Migrate {productname} Proxy from openSUSE Leap Micro 5.5 to Tumbleweed
+:revdate: 2025-10-27
+:page-revdate: {revdate}
+:description: This page describes how to migrate a {productname} Proxy host from openSUSE Leap Micro 5.5 to a fresh openSUSE Tumbleweed installation using the proxy administration tool [command]``mgrpxy``.
+ifeval::[{mlm-content} == true]
+
+:noindex:
+endif::[]
+
+This page describes how to migrate a {productname} Proxy host from openSUSE Leap Micro 5.5 to a fresh openSUSE Tumbleweed installation using the proxy administration tool [command]``mgrpxy``.
+
+[IMPORTANT]
+====
+This guide was tested on Tumbleweed only. There is no known reason it wouldn’t work on other supported bases, but always validate in a test environment before production.
+====
+
+
+
+== Overview of the Proxy Migration Process
+
+You will:
+
+* Save proxy configuration from the old system (including Apache/Squid tuning).
+* Reinstall the host with openSUSE Tumbleweed.
+* Re-register the host using the system reactivation key.
+* Install [command]``mgrpxy`` (and Podman if needed).
+* Restore configuration and run [command]``mgrpxy install podman`` with optional tuning files.
+
+
+== Requirements and Considerations
+
+* Keep the same hostname/FQDN and IP when possible so the server and clients interact with the proxy as before.
```

Comment:
`when possible` is strange here as if the FQDN / IP is not kept the migration cannot work using the next steps

Reply:
I will add the fix in the next batch. I do not think this is breaking (though **possibly** misleading).